doc/source/tutorial/integrate.rst

```diff
@@ -385,12 +385,12 @@ The Python tuple is returned as expected in a reduced amount of time.  All
 optional parameters can be used with this method including specifying
 singularities, infinite bounds, etc.
 
-Ordinary differential equations (:func:`odeint`)
+Ordinary differential equations (:func:`solve_ivp`)
 ------------------------------------------------
```

Comment:
Add more `-` to match the title (restructuredtext requirement)

Reply:
Has been done.